GP03FLMB/GP03FLMB_R00002_ingest.csv

```diff
@@ -1,34 +1,34 @@
+#,/omc_data/whoi/OMC/GP03FLMB/R00002/cg_data/dcl00/CTDMOs_all/CTD*.DAT,GP03FLMB-RIM01-02-CTDMOG060,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes. 
+#mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00002/cg_data/dcl00/CTDMO-G_sn11646/SBE37*11646*.hex,GP03FLMB-RIM01-02-CTDMOG060,recovered_inst,Available,
+#,,GP03FLMB-RIM01-02-CTDMOG061,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes. 
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00002/cg_data/dcl00/CTDMO-G_sn11653/SBE37*11653*.hex,GP03FLMB-RIM01-02-CTDMOG061,recovered_inst,Available,
+#,,GP03FLMB-RIM01-02-CTDMOG062,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes. 
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00002/cg_data/dcl00/CTDMO-G_sn11641/SBE37*11641*.hex,GP03FLMB-RIM01-02-CTDMOG062,recovered_inst,Available,
+#,,GP03FLMB-RIM01-02-CTDMOG063,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes. 
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00002/cg_data/dcl00/CTDMO-G_sn11650/SBE37*11650*.hex,GP03FLMB-RIM01-02-CTDMOG063,recovered_inst,Available,
+#,,GP03FLMB-RIM01-02-CTDMOG064,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes. 
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00002/cg_data/dcl00/CTDMO-G_sn11638/SBE37*11638*.hex,GP03FLMB-RIM01-02-CTDMOG064,recovered_inst,Available,
+#,,GP03FLMB-RIM01-02-CTDMOG065,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes. 
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00002/cg_data/dcl00/CTDMO-G_sn11640/SBE37*11640*.hex,GP03FLMB-RIM01-02-CTDMOG065,recovered_inst,Available,
+#,,GP03FLMB-RIM01-02-CTDMOG066,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes. 
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00002/cg_data/dcl00/CTDMO-G_sn11639/SBE37*11639*.hex,GP03FLMB-RIM01-02-CTDMOG066,recovered_inst,Available,
+#,,GP03FLMB-RIM01-02-CTDMOG067,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes. 
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00002/cg_data/dcl00/CTDMO-G_sn10223/SBE37*10223*.hex,GP03FLMB-RIM01-02-CTDMOG067,recovered_inst,Available,
+#,,GP03FLMB-RIM01-02-CTDMOG068,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes. 
+#mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00002/cg_data/dcl00/CTDMO-G_sn11654/SBE37*11654*.hex,GP03FLMB-RIM01-02-CTDMOG068,recovered_inst,Available,

```

Comment:
I think this line needs to be uncommented.

Reply:
Agreed